lms/djangoapps/ccx/api/v0/serializers.py

```diff
@@ -19,6 +19,7 @@ class CCXCourseSerializer(serializers.ModelSerializer):
     due = serializers.CharField(allow_blank=True)
     max_students_allowed = serializers.IntegerField(source='max_student_enrollments_allowed')
     course_modules = serializers.SerializerMethodField()
+    other_course_settings = serializers.JSONField()
```

Comment:
As it's now, this field is required: https://www.django-rest-framework.org/api-guide/fields/#required @JuanDavidBuitrago